File: src/main/java/com/sjs/lootbotga/game/cards/MerchantValue.java
package com.sjs.lootbotga.game.cards;

public enum MerchantValue implements CardValue{
	TWO (2,5),
	THREE (3,6),
	FOUR (4,5),
	FIVE (5,5),
	SIX (6,2),
	SEVEN (7,1),
	EIGHT (8,1);
	
	private Integer value;
	private Integer amount;

	private MerchantValue(Integer value, Integer amount) {
		this.value = value;
		this.amount  = amount;
	}

	public Integer getValue() {
		return value;
	}

	public Integer getAmount() {
		return amount;
	}
}

File: src/main/kotlin/com/sjs/lootbotga/game/cards/MerchantValue.kt
package com.sjs.lootbotga.game.cards

enum class MerchantValue(private val value: Int, private val amount: Int) : CardValue {
    TWO(2, 5),
    THREE(3, 6),
    FOUR(4, 5),
    FIVE(5, 5),
    SIX(6, 2),
    SEVEN(7, 1),
    EIGHT(8, 1);

    override fun getValue(): Int {
        return value
    }

    override fun getAmount(): Int {
        return amount
    }
}